src/pages/HomePage/AuthForms/Forms.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { Button, Dialog, DialogActions } from '@material-ui/core';
+import {
+  HowToReg as HowToRegIcon,
+  AddToHomeScreen as AddToHomeScreenIcon
+ } from '@material-ui/icons';
+import SignIn from "./Login";
+import Register from "./Register";
+
+const Forms: React.FC = () => {
+  const [open, setOpen] = React.useState(false);
+  const [LoginForm, SetLoginForm] = React.useState(false);
+
+  const trigger = (value: boolean) => {
+    SetLoginForm(value);
+    setOpen(true);
+  };
+
+  // const handleClickOpen = () => {
+  //   setOpen(true);
+  // };
```

Comment:
you can delete commented code ;d

Reply:
Yes, I know, but for some purpose I keep it. :)